components/AutomateBettingForm.tsx

```diff
@@ -68,12 +74,11 @@ export default function SpreadsheetForm() {
     useEffect(() => {
         console.log("Getting stored data")
         const storedUserData = JSON.parse(window.localStorage.getItem('user-data'));
-        console.log("Stored user data", storedUserData);
+        console.log("Stored user data was", storedUserData);
         const storedProcessedData = JSON.parse(window.localStorage.getItem('processed-data'));
-        const storedApiKey = window.localStorage.getItem('api-key');
```

Comment:
as far as I could tell this was loading the stored key from the wrong key (we were setting it in `manifold` elsewhere)